onadata/settings/base.py

```diff
@@ -351,9 +351,12 @@ def skip_suspicious_operations(record):
 DEBUG = env.bool('DJANGO_DEBUG', True)
 
 # Database (i.e. PostgreSQL)
-DATABASES = {
-    'default': env.db(default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)
-}
+DATABASES = {}
+if 'KPI_DATABASE_URL' in env.ENVIRON:
+    DATABASES['default'] = env.db_url(default='KC_DATABASE_URL')
+else:
+    DATABASES['default'] = env.db_url(default='DATABASE_URL')
```

Comment:
I think the syntax should be `env.db_url('DATABASE_URL', default="sqlite:///%s/db.sqlite3" % PROJECT_ROOT)`.